src/packages/DAO/elections.dao.ts

```diff
@@ -51,18 +57,38 @@ export async function startElection(formData: FormData) {
   redirect(`elections/${electionId}`);
 }
 
+async function getElections() {
+  const res = await prisma.election.findMany();
+  if (!res) {
+    throw new NotFound("");
+  }
+  return res;
+}
+async function findByNameOrStatus({ name, status }: FindElection) {
```

Comment:
minor: we usually separate functions at the root level of the file with an empty line, to facilitate paragraph navigation:

```suggestion
}

async function findByNameOrStatus({ name, status }: FindElection) {
```

Reply:
change here [7028497](https://github.com/podcodar/voting-system/pull/67/commits/7028497349e8d13dc2a9fae5c527dc1d51dd085a)